### PR TITLE
chore(deps): update tekton and quay plugin

### PR DESCRIPTION
**Pull request by karthikjeeyar**
## Description

Update tekton and quay plugin to include the latest features and fixes.

## Which issue(s) does this PR fix

Tekton Features:

- Add CVE summary as a column in Pipelinerun list view - [#986](https://github.com/janus-idp/backstage-plugins/issues/986)
- Add signed badge for PipelineRuns that are signed by chains - [#987](https://github.com/janus-idp/backstage-plugins/issues/987)   
- Add view sbom and view logs actions column in the pipelineRun list page - [#988](https://github.com/janus-idp/backstage-plugins/issues/988) 
- Add download logs action - [#878](https://github.com/janus-idp/backstage-plugins/issues/878) 

Quay Fixes:

- security scan column is loading forever for .sbom, .att and .sig tags - [#989](https://github.com/janus-idp/backstage-plugins/issues/989) 
- Vulnerability list page should be sorted by severity - [#990](https://github.com/janus-idp/backstage-plugins/issues/990)  
-  Vulnerability list page should be sorted by severity [#1032](https://github.com/janus-idp/backstage-plugins/issues/1032) 

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

**Comment by invincibleJai** [comment]
Verified the changes and looks good 

Tekton:

image


Quay:

image

File: dynamic-plugins/imports/package.json [hunk 1]
```diff
@@ -25,8 +25,8 @@
     "@janus-idp/backstage-plugin-nexus-repository-manager": "1.4.14",
     "@janus-idp/backstage-plugin-ocm": "3.5.9",
     "@janus-idp/backstage-plugin-ocm-backend": "3.4.10",
-    "@janus-idp/backstage-plugin-quay": "1.4.16",
-    "@janus-idp/backstage-plugin-tekton": "3.1.14",
+    "@janus-idp/backstage-plugin-quay": "1.4.21",
+    "@janus-idp/backstage-plugin-tekton": "3.4.0",
```
[the remark this answers is not in the document]
Comment (karthikjeeyar) [reply dynamic-plugins/imports/package.json]: Updated

**Comment by schultzp2020** [comment]
No longer needed https://github.com/janus-idp/backstage-showcase/pull/899

**Pull request closed without merging**